src/clj/rems/middleware.clj

```diff
@@ -76,7 +76,7 @@
                                  (set/union (roles/get-roles (getx-user-id))
                                             (organizations/get-all-organization-roles (getx-user-id))
                                             (workflow/get-all-workflow-roles (getx-user-id))
-                                            (applications/get-all-application-roles (getx-user-id))))
+                                            (application/get-all-application-roles (getx-user-id))))
```

Comment:
Let's refactor this to use `rems.service.user/get-roles` which would use `rems.service.cache/get-roles` and do this whole thing.